src/input.rs

```diff
@@ -107,6 +107,36 @@ pub type KeyBinding = Binding<Key>;
 /// Bindings that are triggered by a mouse button
 pub type MouseBinding = Binding<MouseButton>;
 
+impl Default for Action {
+    fn default() -> Action {
+        Action::PasteSelection
+    }
+}
+
+impl Default for KeyBinding {
+    fn default() -> KeyBinding {
+        KeyBinding {
+            mods: Default::default(),
+            action: Action::Esc(String::new()),
+            mode: TermMode::NONE,
+            notmode: TermMode::NONE,
+            trigger: Key::A,
+        }
+    }
+}
+
+impl Default for MouseBinding {
+    fn default() -> MouseBinding {
+        MouseBinding {
+            mods: Default::default(),
+            action: Action::Esc(String::new()),
+            mode: TermMode::NONE,
+            notmode: TermMode::NONE,
+            trigger: MouseButton::Left,
+        }
+    }
+}
```

Comment:
It's absolutely correct that the implementation of the `Action` trait was incorrect. I've decided to use that for temporary testing since it doesn't have many side effects, but forgot to create a more proper implementation later.

I've added a `None` action to the enum, this makes the Default implementation useful and also allows easily overriding the default bindings (also added docs for that to the yaml).

A ConfigDefault trait isn't usefull precisely because it's not Default. The great thing about the Default trait is that it can be used by everyone, which includes serde. This makes deserialization a lot less verbose. Of course Default should only be implemented when the implementation actually makes sense.